CHANGELOG.md

```diff
@@ -31,6 +31,7 @@
 - fix calling field resolver without providing resolver class to `buildSchema`
 - fix generated TS union type for union type of object type classes extending themselves (#587)
 - fix using shared union and interface types in multiple schemas when `resolveType` is used
+- fix `@Directive()` cannot be extends by subclasses 
```

Comment:
This entry it's for the "unreleased" section as merging this PR won't overwrite `1.0.0-rc.1` 😉